src/test/java/lt/andriaus/hangman/usecase/GameInRamManagerTest.java

```diff
@@ -0,0 +1,56 @@
+package lt.andriaus.hangman.usecase;
+
+import lt.andriaus.hangman.domain.Game;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class GameInRamManagerTest {
+    GameInRamManager gameInRamManager;
+    @Mock
+    Game game;
+    @Mock
+    RamDatabase<String> wordDB;
+    @Mock
+    RamDatabase<Game> gameDB;
+
+
+    @BeforeEach
+    void setup(){
+        gameInRamManager = new GameInRamManager(wordDB, gameDB);
+    }
+
+    @Test
+    void shouldCreateGame() {
+        Map<Integer, String> wordMap = new HashMap<>();
+        wordMap.put(0, "HELLO");
```

Comment:
`Map<Integer, String> wordMap = new HashMap<>();
wordMap.put(0, "HELLO");` -> `Map<Integer, String> wordMap = Map.of(0, "HELLO");`